hw10.py

```diff
@@ -0,0 +1,54 @@
+# 1. До завдання, в якому ви реалізовували телефонну книгу, додати валідацію номера телефону за допомогою RegEx.
+# Врахувати можливість декількох форматів: +380XXXXXXXXX, 380XXXXXXXXX, 0XXXXXXXXX
+import json
+import re
+enter = input('''You can see all contacts(stats), all names(list), someone\'s number(show) or (add), (delete) contacts.
+Enter your command: ''')
+if enter == 'add':
+    name = input('Enter name to add: ')
+    phone = input('Enter phone to add: ')
+    match1 = re.search('\+380\d{9}', phone)
```

Comment:
For this case it would be better to use `re.fullmatch` since use can enter something like `hello+380951234567` and it will be valid in your case